File: 351001/Rudzenia/RW.Application/DTOs/Response/ArticleResponseTo.cs
using System.Text.Json.Serialization;

namespace RW.Application.DTOs.Response;

public class ArticleResponseTo
{
    [JsonPropertyName("id")]
    public long Id { get; set; }

    [JsonPropertyName("authorId")]
    public long AuthorId { get; set; }

    [JsonPropertyName("title")]
    public string Title { get; set; } = string.Empty;

    [JsonPropertyName("content")]
    public string Content { get; set; } = string.Empty;

    [JsonPropertyName("created")]
    public DateTime Created { get; set; }

    [JsonPropertyName("modified")]
    public DateTime Modified { get; set; }
}

File: 351004/Beliaev/restapp/src/main/java/com/example/restapp/dto/response/ArticleResponseTo.java
package com.example.discussion.dto.response;

import lombok.Data;
import java.time.LocalDateTime;
import java.util.List;

@Data
public class ArticleResponseTo {
    private Long id;
    private Long authorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
    private List<Long> stickerIds;
}